tests/components/generic/test_init.py

```diff
@@ -0,0 +1,37 @@
+"""Define tests for the generic general setup."""
```

Comment:
Stale docstring

Reply:
Not sure what a good docstring is here. I copied the format from the next integration up on the list (gdacs)
https://github.com/home-assistant/core/blob/18e8a3b185c66c4870dcc5f7a9662b8ba9ddc883/tests/components/gdacs/test_init.py#L1.
But I agree it sounds weird. How about:
```suggestion
"""Define tests for the generic (IP camera) integration."""
```

Reply:
I now read this back and I actually have no idea why I said this 😂